addon/components/simple-typeahead/template.hbs

```diff
@@ -1,16 +1,25 @@
 <div class="input">
-  <input
+  {{input
     type='text'
-    value={{value}}
-    placeholder={{placeholder}}
-    oninput={{action 'input'}}
-    onkeydown={{action 'keydown'}}
-  />
+    spellcheck=false
+    value=value
+    placeholder=placeholder
+    input=(action 'input')
+    keyDown=(action 'keydown')
+  }}
 
-  <button title='clear input' class='clear' disabled={{valueIsEmpty}} {{action 'clear'}}>&#10005;</button>
+  {{#if showReset}}
+    <i title='clear input' class='clear' {{action 'clear'}}>
```

Comment:
I also see that you removed the ability to hide the clear button all together, in lieu of just disabling it. I think keeping that ability makes sense as someone may not want to show the clear button at all.